compiler_opt/es/blackbox_optimizers.py

```diff
@@ -75,6 +78,14 @@ class CurrentPointEstimate(enum.Enum):
   AVERAGE = 2
 
 
+@gin.constants_from_enum(module='blackbox_optimizers')
+class Algorithm(enum.Enum):
+  MONTE_CARLO = 1
+  TRUST_REGION = 2
+  SKLEARN_REGRESSION = 3
+
+
+@gin.constants_from_enum(module='blackbox_optimizers')
```

Comment:
why not also @gin - mark EstimatorType, GrandientType, and RegressionType? might as well...